examples/chart/teleport-kube-agent/templates/deployment.yaml

```diff
@@ -35,9 +40,37 @@ spec:
       labels:
         app: {{ .Release.Name }}
     spec:
-      {{- if .Values.affinity }}
+      {{- if or .Values.affinity (gt (int .Values.highAvailability.replicaCount) 1) }}
```

Comment:
Should this be checking against `.replicaCount` instead as that aggregates both values?

Reply:
Yes, you're right. Let me make that change.

Do we need to update the statefulset.yaml as well? I didn't do that so we may want to.

Reply:
The `StatefulSet` is used when persistent storage is enabled so yes, please update that one too!

Reply:
@webvictim Finally back to this and will work to get it merged. Stateful set is updated as well now.